core/pkg/opaprocessor/processorhandlerutils.go

```diff
@@ -46,7 +47,7 @@ func (opap *OPAProcessor) updateResults() {
 
 	// set result summary
 	// map control to error
-	controlToInfoMap := mapControlToInfo(opap.ResourceToControlsMap, opap.InfoMap)
+	controlToInfoMap := mapControlToInfo(opap.ResourceToControlsMap, opap.InfoMap, opap.Report.SummaryDetails.Controls)
```

Comment:
Call the function and not the object ->`opap.Report.SummaryDetails.GetControls()`

Reply:
There is no such method. There is ListControls() but that gives a list of controlsummary which I cannot access in a map way by ID (the way it's done with controlsummaries)